mora/service/engagement.py

```diff
@@ -216,7 +211,10 @@ def get_employee_id(effect):
         return effect['relationer']['tilknyttedebrugere'][-1]['uuid']
 
     def get_unit_id(effect):
-        return effect['relationer']['tilknyttedeenheder'][-1]['uuid']
+        try:
+            return effect['relationer']['tilknyttedeenheder'][-1]['uuid']
+        except (KeyError, IndexError):
+            return None
```

Comment:
Could you add a comment explaining which cases trigger this? It's not obvious to me.

Reply:
This is relevant for 'Leave' as we only have relations to users, not org units.

Reply:
Precisely; it'd be nice to have a line noting that in the code.